src/test/java/uk/gov/justice/digital/delius/integration/secure/StaffResource_StaffDetailsAPITest.java

```diff
@@ -134,4 +139,74 @@ public void retrievingStaffDetailsByUsernameReturn404WhenUserDoesNotExist() {
                 .then()
                 .statusCode(404);
     }
+
+    @Test
+    public void retrieveStaffDetailsForMultipleUsers() {
+
+        val staffDetails = given()
+                .auth()
+                .oauth2(validOauthToken)
+                .contentType(APPLICATION_JSON_VALUE)
+                .body(getUsernames(Set.of("sheilahancocknps", "JimSnowLdap")))
+                .when()
+                .post("staff/list")
+                .then()
+                .statusCode(200)
+                .extract()
+                .body()
+                .as(StaffDetails[].class);
+
+        StaffDetails jimSnowUserDetails = Arrays.stream(staffDetails).filter(s -> s.getUsername().equals("JimSnowLdap")).collect(Collectors.toList()).get(0);
```

Comment:
very minor, but maybe use var and findFirst()

Reply:
agreed and changed